release/ray_release/scripts/ray_bisect.py

```diff
@@ -28,30 +28,37 @@ def main(test_name: str, passing_commit: str, failing_commit: str) -> None:
         )
         return
     commit_lists = _get_commit_lists(passing_commit, failing_commit)
-    blamed_commit = _bisect(test, commit_lists)
+    blamed_commit = _bisect(test, commit_lists, 3)
     logger.info(f"Blamed commit found for test {test_name}: {blamed_commit}")
 
 
-def _bisect(test: Test, commit_list: List[str]) -> str:
+def _bisect(test: Test, commit_list: List[str], concurrency: int) -> str:
     while len(commit_list) > 2:
         logger.info(
             f"Bisecting between {len(commit_list)} commits: "
             f"{commit_list[0]} to {commit_list[-1]}"
         )
-        middle_commit_idx = len(commit_list) // 2
-        middle_commit = commit_list[middle_commit_idx]
-        is_passing = _run_test(test, [middle_commit])[middle_commit] == "passed"
-        if is_passing:
-            commit_list = commit_list[middle_commit_idx:]
-        else:
-            commit_list = commit_list[: middle_commit_idx + 1]
+        idx_to_commit = {}
+        for i in range(1, concurrency+1):
+            idx = i * len(commit_list) // (concurrency+1)
```

Comment:
maybe `len(commit_list) * i // (concurrency+1)` so that it is clearer that `len(commit_list)` is the base thing that we are trying to split here?